woocommerce/woocommerce
# 31824 - Include order ID in the error log message

### All Submissions:


### Changes proposed in this Pull Request:

Closes #31824.

### How to test the changes in this Pull Request:

The goal is to trigger an error during the Order Saving process.

Here is how I tested it in the code:

1. In this file, (`plugins/woocommerce/includes/abstracts/abstract-wc-order.php`) inside the `try` block (around line 178), place this code:
```
throw new ErrorException( 'test message', 0, 1, __FILE__, 1 );
```
2. Go into any Order in the WP Admin, then click the "Update" button.
3. Visit Status > Logs (`wp-admin/admin.php?page=wc-status&tab=logs`) and ensure the errors are logged with a message like:
"2022-03-16T14:06:29+00:00 ERROR Error saving order ID 86."
or
"2022-03-16T14:06:29+00:00 ERROR Error saving order ID (no ID)."

### Other information:




### Changelog entry

> Tweak - Updated Order save error messages to include the ID.

### FOR PR REVIEWER ONLY:
